src/lib/backpack-api.js

```diff
@@ -0,0 +1,19 @@
+const getBackpackContents = ({
+    host,
+    username,
+    token,
+    limit,
+    offset
+}) => fetch(`${host}${username}?limit=${limit}&offset=${offset}`, {
```

Comment:
We have been using `xhr` for requests: https://github.com/LLK/scratch-gui/blob/0baf6852fcc0ff42ff39a94e27e39b17378d757b/src/components/costume-canvas/costume-canvas.jsx#L79. Even though this is probably here temporarily, I think we should use the same method for requests everywhere.